packages/nodes-base/nodes/Wait/Wait.node.ts

```diff
@@ -346,6 +347,7 @@ export class Wait extends Webhook {
 				},
 				default: '',
 				description: 'The date and time to wait for before continuing',
+				required: true,
```

Comment:
A deeper fix would be having this param type fail when `required: true` and still using a `''` default for all nodes. 

Since this is a rare edge case that deeper change might have unexpected side effects, I'd rather prevent this at the `nodes-base` and `core` levels for now.